dotCMS/src/main/java/com/dotcms/browser/BrowserAPI.java

```diff
@@ -0,0 +1,585 @@
+			List<String> extensions, boolean showWorking, boolean showArchived, boolean noFolders,
+			boolean onlyFiles, String sortBy, boolean sortByDesc)
+			throws DotSecurityException, DotDataException {
+
+    	return getFolderContent(user, folderId, offset, maxResults, filter, mimeTypes, extensions, showWorking, showArchived, noFolders, onlyFiles, sortBy, sortByDesc, false, 0);
+
+    }
+    @Deprecated
+    public Map<String, Object> getFolderContent(User user, String folderId,
+			int offset, int maxResults, String filter, List<String> mimeTypes,
+			List<String> extensions, boolean showWorking, boolean showArchived, boolean noFolders,
+			boolean onlyFiles, String sortBy, boolean sortByDesc, long languageId)
+			throws DotSecurityException, DotDataException {
+
+    	return getFolderContent(user, folderId, offset, maxResults, filter, mimeTypes, extensions, showWorking, showArchived, noFolders, onlyFiles, sortBy, sortByDesc, false, languageId);
+
+    }
+    
+    protected class WfData {
+        
+        List<WorkflowAction> wfActions = new ArrayList<WorkflowAction>();
+        boolean contentEditable = false;
+        List<Map<String, Object>> wfActionMapList = new ArrayList<Map<String, Object>>();
+        
+        boolean skip=false;
+        
+        public WfData(Contentlet contentlet, List<Integer> permissions, User user, boolean showArchived) throws DotStateException, DotDataException, DotSecurityException {
+            
+            
+            if(null==contentlet) {
+                return;
+            }
+
+            wfActions = APILocator.getWorkflowAPI().findAvailableActions(contentlet, user, WorkflowAPI.RenderMode.LISTING);
+            
+            if (permissionAPI.doesUserHavePermission(contentlet, PermissionAPI.PERMISSION_WRITE, user) && contentlet.isLocked()) {
+                String lockedUserId = APILocator.getVersionableAPI()
+                        .getLockedBy(contentlet);
+                if (user.getUserId().equals(lockedUserId)) {
+                    contentEditable = true;
+                } else {
+                    contentEditable = false;
+                }
+            } else {
+                contentEditable = false;
+            }
+            
+
+
+
+            if (permissions.contains(PERMISSION_READ)) {
+                if (!showArchived && contentlet.isArchived()) {
+                    skip=true;
+                    return;
+                }
+                final boolean showScheme = (wfActions!=null) ?  wfActions.stream().collect(Collectors.groupingBy(WorkflowAction::getSchemeId)).size()>1 : false;
+                
+                for (final WorkflowAction action : wfActions) {
+
+					WorkflowScheme wfScheme = APILocator.getWorkflowAPI().findScheme(action.getSchemeId());
+                    Map<String, Object> wfActionMap = new HashMap<String, Object>();
+                    wfActionMap.put("name", action.getName());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "name" appears 8 times in this file; the first occurrence is on line 125](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5337824)